include/sys/dsl_pool.h

```diff
@@ -115,6 +117,7 @@ typedef struct dsl_pool {
 	kcondvar_t dp_spaceavail_cv;
 	uint64_t dp_dirty_pertxg[TXG_SIZE];
 	uint64_t dp_dirty_total;
+	hrtime_t dp_last_smooth;
```

Comment:
doc comment suggestion, if I understand the code correctly:

> The point in time time after which the write smoothing mechanism has no effect.

If that interpretation of the variable's role is correct, I'd prefer a different name.
For example, `dp_stop_write_smooth_after`.

Reply:
This isn't a variable described in doc.
This is variable to store a time of last smoothing.